src/saltext/azurerm/utils/azurerm.py

```diff
@@ -332,7 +335,6 @@ def get_identity_credentials(**kwargs):
     for keyword, value in kwarg_map.items():
         if kwargs.get(keyword):
             os.environ[value] = kwargs[keyword]
-
```

Comment:
**suggestion (non-blocking):** unless a `pre-commit` check like `black` is adding/removing empty lines, it's best to refrain from unrelated whitespace changes in PRs.